integreat_cms/cms/linklists.py

```diff
@@ -59,6 +59,21 @@ class EventTranslationLinklist(ContentTranslationLinklist):
     """
 
     model = EventTranslation
+    html_fields = ["content"]
```

Comment:
```suggestion
```
This is not required since `EventTranslationLinklist` inherits from `ContentTranslationLinklist` which already sets this field.